spaghetto/bin/src/Main.purs

```diff
@@ -403,56 +454,68 @@ main =
           RegistryInfo args -> do
             env <- mkRegistryEnv
             void $ runSpago env (Registry.info args)
-          Install args@{ packages, selectedPackage, ensureRanges } -> do
+          Install args@{ packages, selectedPackage, ensureRanges, jsonErrors } -> do
             { env, fetchOpts } <- mkFetchEnv { packages, selectedPackage, ensureRanges }
             -- TODO: --no-fetch flag
             dependencies <- runSpago env (Fetch.run fetchOpts)
             env' <- runSpago env (mkBuildEnv args dependencies)
-            let options = { depsOnly: true, pursArgs: List.toUnfoldable args.pursArgs }
+            let options = { depsOnly: true, pursArgs: List.toUnfoldable args.pursArgs, jsonErrors }
```

Comment:
Should we drop `json-errors` from `InstallArgs` and just set this to `false`? Likewise for Bundle, Run, and Test?

Reply:
Agreed. As noted above, we should generally try to minimise the amount of flags exposed in any command, since they all add a burden when trying to use a specific command.